table/get_context.cc

```diff
@@ -182,7 +182,8 @@ bool GetContext::SaveValue(const ParsedInternalKey& parsed_key,
   assert(matched);
   assert((state_ != kMerge && parsed_key.type != kTypeMerge) ||
          merge_context_ != nullptr);
-  if (ucmp_->Equal(parsed_key.user_key, user_key_)) {
+  if (ucmp_->Equal(parsed_key.user_key, user_key_) ||
+      ucmp_->CompareWithoutTimestamp(parsed_key.user_key, user_key_) == 0) {
```

Comment:
Is `ucmp_->CompareWithoutTimestamp(parsed_key.user_key, user_key_) == 0` just enough? If there is not timestamp, then they are the same thing.
If there is an Equal(), we may need EqualWithoutTimestamp()..
Or we can say that, we are not going to support byte-wide different keys to be same if users want to use timestamp feature.

Reply:
Yes you are right. Will fix.